contributors/kosuke316.md

```diff
@@ -0,0 +1 @@
+Correction of japanese mistakes.
```

Comment:
Hey @kosuke316 thanks for the PR, we don't maintain this style of contributor log so need this to be removed.

Reply:
sorry, I got it!